pro-duck-store/src/model/DrawerListItemProp.tsx

```diff
@@ -0,0 +1,4 @@
+export interface ListItemProps {
+    title: string,
+    Icon?: any
```

Comment:
Try to be consistent with naming conventions. Unless it's not a class, it should start with lower case.